From 3b8e732d85578aa3a141552647d0ebbff31ede3d Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Wed, 4 Dec 2013 11:11:26 +0000
Subject: [PATCH 101/351] ASoC: arizona: Add support for 768kHz DMIC operation

The new IPs supports a new lower frequency 768kHz DMIC operation
add support for this into the OSR control.

Change-Id: I355a9a65744da867d0833059586d60f568f2f8b4
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Signed-off-by: Nikesh Oswal <Nikesh.Oswal@wolfsonmicro.com>
---
 sound/soc/codecs/arizona.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/arizona.c b/sound/soc/codecs/arizona.c
index 95f1096..d568957 100644
--- a/sound/soc/codecs/arizona.c
+++ b/sound/soc/codecs/arizona.c
@@ -642,7 +642,7 @@ const struct soc_enum arizona_in_hpf_cut_enum =
 EXPORT_SYMBOL_GPL(arizona_in_hpf_cut_enum);
 
 static const char * const arizona_in_dmic_osr_text[] = {
-	"1.536MHz", "3.072MHz", "6.144MHz",
+	"1.536MHz", "3.072MHz", "6.144MHz", "768kHz",
 };
 
 const struct soc_enum arizona_in_dmic_osr[] = {
-- 
1.7.9.5

